exchanges/kucoin/kucoin_websocket.go

```diff
@@ -38,16 +38,18 @@ const (
 	publicBullets  = "/v1/bullet-public"
 	privateBullets = "/v1/bullet-private"
 
-	// spot channels
-	marketTickerChannel           = "/market/ticker"            // /market/ticker:{symbol},...
-	marketSnapshotChannel         = "/market/snapshot"          // /market/snapshot:{symbol},...
+	// Spot channels
+	marketAllTickersChannel       = "/market/ticker:all"
+	marketTickerChannel           = "/market/ticker"            // /market/ticker:{symbol},{symbol}...
+	marketSymbolSnapshotChannel   = "/market/snapshot"          // /market/snapshot:{symbol}
+	marketSnapshotChannel         = "/market/snapshot"          // /market/snapshot:{market} <--- market represents a currency
 	marketOrderbookChannel        = "/market/level2"            // /market/level2:{symbol},...
 	marketOrderbookDepth5Channel  = "/spotMarket/level2Depth5"  // /spotMarket/level2Depth5:{symbol},...
 	marketOrderbookDepth50Channel = "/spotMarket/level2Depth50" // /spotMarket/level2Depth50:{symbol},...
-	marketCandlesChannel          = "/market/candles"           // /market/candles:{symbol}_{interval},...
-	marketMatchChannel            = "/market/match"             // /market/match:{symbol},...
-	indexPriceIndicatorChannel    = "/indicator/index"          // /indicator/index:{symbol},...
-	markPriceIndicatorChannel     = "/indicator/markPrice"      // /indicator/markPrice:{symbol},...
+	marketCandlesChannel          = "/market/candles"           // /market/candles:{symbol}_{interval}
+	marketMatchChannel            = "/market/match"             // /market/match:{symbol},{symbol}...
+	indexPriceIndicatorChannel    = "/indicator/index"          // /indicator/index:{symbol0},{symbol1}..
+	markPriceIndicatorChannel     = "/indicator/markPrice"      // /indicator/markPrice:{symbol0},{symbol1}...
```

Comment:
Hey @samuael  👋 

**This comment includes one serious issue below**

It seems from this section that [these changes](https://github.com/thrasher-corp/gocryptotrader/commit/b41fe276846bde36fae6da4354f7a1381191d70a#diff-7860ba044baa8916bee4b6d47c3b0dc647baf693953a74ce39da9c3826b39e57)
from #1579 were dropped by your recent merge of master. I think you've taken your side of the merge conflict only, maybe.

Notably:
* Duplicate channel names have been removed (e.g. `marketSymbolSnapshotChannel` vs `marketSnapshotChannel`
* marketAllTickersChannel has been removed
* symbol comments have been made consistent, so they just say `,...` instead of mixing symbol0, symbol etc.
* Candles channel comment was updated
* Same changes to comments below for Futures channels
* 🚨  **Ticker v1 has been removed. We don't use it. Instead v2 was renamed to just be** 
`futuresTickerChannel`. This is fatal because it means if merged, we'll be using the v1 endpoint you've re-added 🚨

Reply:
Selam 🕊️  @gbjk

Thank you for updating me.

I think I satisfied all 6 points you raised above and published the change with the following commit.
I hope to get your comments if there is any thing I should address before we merge this branch.

Reply:
I already have tested the websocket data handling at runtime; You could do so and let me know if there might be any issues you might find.